app/managers.py

```diff
@@ -0,0 +1,39 @@
+import sqlite3
+
+from models import Actor
+
+
+class ActorManager:
+    def __init__(self) -> None:
+        self._connect = sqlite3.connect("../cinema.sqlite")
+        self.table_name = "actors"
+
+    def create(self, first_name: str, last_name: str) -> None:
+        self._connect.execute(
+            f"INSERT INTO {self.table_name}"
+            f" (first_name, last_name) VALUES (?, ?)",
+            (first_name, last_name)
+        )
+        self._connect.commit()
+
+    def all(self) -> list:
+        actor_cursor = self._connect.execute(
+            f"SELECT * FROM {self.table_name}"
+        )
+        return [Actor(*row) for row in actor_cursor]
+
+    def update(self, update_first_name: str, update_last_name: str,
+               id_to_update: int) -> None:
+        self._connect.execute(
+            f"UPDATE {self.table_name}"
+            f" SET first_name=?, last_name=? WHERE id=?",
+            (id_to_update, update_first_name, update_last_name)

```

Comment:
The `update` method has the parameters in the wrong order. The `id_to_update` should be the last parameter in the SQL query to ensure the correct record is updated. Please correct the order to `(update_first_name, update_last_name, id_to_update)`.